sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1863,15 +1863,26 @@ public partial class FacetResult : System.Collections.Generic.IEnumerable<System
     {
         internal FacetResult() { }
         public long? Count { get { throw null; } }
+        public object From { get { throw null; } }
         public object this[string key] { get { throw null; } }
         public System.Collections.Generic.IEnumerable<string> Keys { get { throw null; } }
         int System.Collections.Generic.IReadOnlyCollection<System.Collections.Generic.KeyValuePair<System.String,System.Object>>.Count { get { throw null; } }
+        public object To { get { throw null; } }
+        public Azure.Search.Documents.Models.FacetType Type { get { throw null; } }
```

Comment:
Thinking about this a little more, I should change this to `public Azure.Search.Documents.Models.FacetType FacetType` instead of just `Type`.